packages/components/data-resource/s3-endpoint-type.tsx

```diff
@@ -60,15 +60,13 @@ export const S3EndPointType: React.FC<S3EndpointTypeProps> = (props) => {
   };
 
   const getSecrets = React.useCallback(
-    (e) =>
+    (resorce) =>
```

Comment:
though it is being changed [here](https://github.com/red-hat-storage/mcg-ms-console/pull/39).
resorce -> resource